samples/crypto/aes_cbc/boards/nrf9160dk_nrf9160ns.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_BUILD_WITH_TFM=y
+CONFIG_TFM_PARTITION_AUDIT_LOG=n
+CONFIG_MBEDTLS_RSA_C=n
+CONFIG_TFM_BL2=n
+CONFIG_PM_PARTITION_SIZE_BL2=0x0
```

Comment:
Disabling audit is because we don't need it for the example.

RSA is here for a temporary fix (will be reverted later when RSA is supported)

BL2 configuration Since these are negating configurations you would need to put this in e.g. Kconfig.nrf (changing the defaults) which would likely make a wrong precedence down the line.... The example-specific alternative is to make some kind of overlay, but that is equally noisy...

Reply:
Maybe have a single `common.conf` in `samples/crypto` and point to that in each `testcase.yaml`, in order to avoid the duplication?

Reply:
Keeping as-is for now